pom.xml

```diff
@@ -37,12 +37,12 @@
 	</licenses>
 	
 	<properties>
-        <vaadin.version>14.8.20</vaadin.version>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <vaadin.version>24.0.0.alpha5</vaadin.version>
```

Comment:
We should maintain Vaadin 14 and JDK 8, so that the addon is compatible with Vaadin 14 to 24. If possible, it would be interesting to add a maven profile for Vaadin 24.